yt_dlp/extractor/la7.py

```diff
@@ -57,3 +63,134 @@ def _real_extract(self, url):
             'thumbnail': self._og_search_thumbnail(webpage, default=None),
             'ie_key': 'Kaltura',
         }
+
+
+class LA7PodcastEpisodeIE(InfoExtractor):
+    IE_NAME = 'la7.it:pod:episode'
+    _VALID_URL = r'''(?x)(https?://)?
+        (?:www\.)?la7\.it/[^/]+/podcast/([^/]+-)?(?P<id>\d+)'''
+
+    _TESTS = [{
+        'url': 'https://www.la7.it/voicetown/podcast/la-carezza-delle-memoria-di-carlo-verdone-23-03-2021-371497',
+        'md5': '7737d4d79b3c1a34b3de3e16297119ed',
+        'info_dict': {
+            'id': '371497',
+            'ext': 'mp3',
+            'title': '"La carezza delle memoria" di Carlo Verdone',
+            'description': 'md5:5abf07c3c551a687db80af3f9ceb7d52',
+            'thumbnail': 'https://www.la7.it/sites/default/files/podcast/371497.jpg',
+            'upload_date': '20210323',
+        },
+    }, {
+        # embed url
+        'url': 'https://www.la7.it/embed/podcast/371497',
+        'only_matching': True,
+    }, {
+        # date already in the title
+        'url': 'https://www.la7.it/propagandalive/podcast/lintervista-di-diego-bianchi-ad-annalisa-cuzzocrea-puntata-del-1932021-20-03-2021-371130',
+        'only_matching': True,
+    }, {
+        # title same as show_title
+        'url': 'https://www.la7.it/otto-e-mezzo/podcast/otto-e-mezzo-26-03-2021-372340',
+        'only_matching': True,
+    }]
+
+    def _extract_infos(self, webpage, video_id=None):
+        if not video_id:
+            video_id = self._search_regex(
+                r'data-nid=([\'"])(?P<vid>\d+)\1',
+                webpage, 'video_id', group='vid')
+
+        media_url = self._search_regex((
+            r'src:\s*([\'"])(?P<url>.+?mp3.+?)\1',
+            r'data-podcast=([\'"])(?P<url>.+?mp3.+?)\1'),
+            webpage, 'media_url', group='url')
+        ext = determine_ext(media_url)
+        formats = [{
+            'url': media_url,
+            'format_id': ext,
+            'ext': ext,
+        }]
+
+        title = self._html_search_regex((
+            r'<div class="title">(?P<title>.+?)</',
+            r'<title>(?P<title>[^<]+)</title>',
+            r'title:\s*([\'"])(?P<title>.+?)\1'),
+            webpage, 'title', group='title')
+
+        description = self._html_search_regex((
+            r'<div class="description">(.+?)</div>',
+            r'<div class="description-mobile">(.+?)</div>',
+            r'<div class="box-txt">([^<]+?)</div>',
+            r'<div class="field-content"><p>(.+?)</p></div>'),
+            webpage, 'description', default=None) or self._html_search_meta(
+                'description', webpage)
```

Comment:
I think this is much more readable, no? Same for others as well
```suggestion
        description = (
            self._html_search_regex(
                (r'<div class="description">(.+?)</div>',
                 r'<div class="description-mobile">(.+?)</div>',
                 r'<div class="box-txt">([^<]+?)</div>',
                 r'<div class="field-content"><p>(.+?)</p></div>'),
                webpage, 'description', default=None)
            or self._html_search_meta('description', webpage))
```